gemini/evaluation/score_and_select_models_rapid_evaluation_sdk.ipynb

```diff
@@ -1,842 +1,803 @@
 {
```

Comment:
<p>"Gen AI Eval Service SDK'</p><p> </p><p>We also need to rename this tutorial to "Compare models". After changing filenames, please update the hyperlinks above </p><p> </p><p>See http://shortn/_SbzHwiTACn</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>